<!--
 Copyright 2022 Shadow Robot Company Ltd.
 This program is free software: you can redistribute it and/or modify it
 under the terms of the GNU General Public License as published by the Free
 Software Foundation version 2 of the License.
 This program is distributed in the hope that it will be useful, but WITHOUT
 ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
 FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
 more details.
 You should have received a copy of the GNU General Public License along
 with this program. If not, see <http://www.gnu.org/licenses/>.
-->

<launch>
    <!-- Launches the hazard light node. -->

    <!-- The status publish rate. Default is 1 as the topic only publishes connection and current light status.
         It is not safety critical. -->
    <arg name="publishing_rate" default="1"/>

    <node name="sr_hazard_light" pkg="sr_hazard_light" type="sr_hazard_light_node" output="screen">
        <param name="publishing_rate" value="$(arg publishing_rate)"/>
    </node>

</launch>
